.github/workflows/stylecheck-prs.yml

```
@@ -0,0 +1,25 @@
+name: stylecheck
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  stylecheck:
+    runs-on: ubuntu-latest
+    steps:
```

This action's GH page says the following:
> Due to the current token permissions, this Action CAN NOT post annotations to PRs from forked repositories.

The following will restrict this job to run only on non-forked PRs.

```suggestion
    if: github.event.pull_request.head.repo.full_name == github.repository
    steps:
```